packages/socket-mode/package.json

```diff
@@ -48,7 +48,7 @@
   },
   "dependencies": {
     "@slack/logger": "^4",
-    "@slack/web-api": "^7.0.1",
+    "@slack/web-api": "^7",
```

Comment:
😅 Ahha versioning dependencies instead of sub dependencies was confusing me, but it does make sense that setting a specific minimum is the better for ensuring the updates are used! I might've been assuming fresh installs was a safe bet here...